public/electron.js

```diff
@@ -78,6 +101,14 @@ function createWindow () {
     // when you should delete the corresponding element.
     mainWindow = null
   })
+
+  mainWindow.on("close", (e) => {
```

Comment:
Maybe we should show a message to the user saying that the application is still running on the background (with a checkbox to stop showing this message).

Reply:
done!